File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        c=0
        for i in range(len(nums)):
            for j in range(len(nums)):
                if(nums[i]==nums[j] and i!=j):
                    c=1
        if c==1:
            return "true"
        else:
            return "false"

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] a1 = new int[nums.length];
        int[] a2 = new int[nums.length];
        a1[0]=nums[0];
        for(int i= 1;i<nums.length ; i++){
            a1[i]=a1[i-1]*nums[i];
        }
        int x=nums.length-1;
        a2[x]=nums[x];
        for(int i= nums.length-2;i>=0 ; i--){
            a2[i]=a2[i+1]*nums[i];
        }
        int[] a3 = new int[nums.length];
        a3[0]=a2[1];
        a3[x]=a1[x-1];
        for(int i=1;i<nums.length-1;i++){
            a3[i]=a1[i-1]*a2[i+1];
        }
        return a3;


    }
}  
